docs/src/main/sphinx/release/release-358.md

```diff
@@ -0,0 +1,40 @@
+# Release 358 (XX May 2021)
+
+## General
+
+* Support any query in {doc}`/sql/show-stats`. ({issue}`8026`)
+* Improve performance of complex queries involving joins and `TABLESAMPLE`. ({issue}`8094`)
+* Improve performance of `ORDER BY ... LIMIT` queries on sorted data. ({issue}`6634`)
+* Reduce worker graceful shutdown time. ({issue}`8149`)
+* Fix query failure when one of join build side's columns is of non-orderable type. ({issue}`7723`)
```

Comment:
Fix query failure when a build side column used in a join uses a non-orderable type.

Reply:
I'm actually thinking we should rephrase in a way that doesn't say "build side". That's an internal implementation concept, not something users can reason about.